### ISSUE TITLE
Question about the repo management

### ISSUE THREAD

**Issue by robotman3000** [marcelloc/Unofficial-pfSense-packages#32]
Hi @marcelloc, I have made a fork of this package repo and I was wondering what version of FreeBSD and the pkg tools you use to update the packages when changes are made. Are you using the legacy pkg_* tools or the new pkgng tools?

**Comment by forid786** [comment]
Hi @robotman3000 I realised that you already forked a repo and have managed to compile V5. How is it going? Have you got it working yet? I gave it a quick spin but seems like E2G5 is still not appearing. The conf file may need some work - ended up restoring my backup, virtualising pfSense is doing me wonders!

**Comment by robotman3000** [comment]
Yes, I have sorta gotten it working. It will build and run but I've run into issues with illegal instruction errors and haven't had the time to check if its been fixed yet. I did create an issue over at the e2guardian repo about it though.

**Comment by marcelloc** [comment]
Get binaries from my repo if you want. All issues we reported are fixed now.
I've also included the transparent proxy options on gui
